assets/references.bibtex

```diff
@@ -0,0 +1,189 @@
+@article{ewels2016multiqc,
```

Comment:
Looks like this is a bibtex file for rnaseq pipeline? Dupradar, Hisat2, STAR aren't used in the workflow at all :-)

Reply:
yes right, I'll update the bibtex file to include the references for this pipeline in the report, I forgot about this completely!